### DISCUSSION TITLE
Bed leveling

### DISCUSSION THREAD

**Discussion by Hardmanpr** [mriscoc/Ender3V2S1#1301] [General]
I installed a bltouch with the spriter extruder then I installed the 2.1.3 MRiscoC 20240125 -01:18. I include de G29 on cura but when I print the printer just make the homing but not check the bed and I select 5 mesh point Y & Z I think maybe I forget something to configure.
Thank in advance
Att Raul
![IMG_3996](https://github.com/mriscoc/Ender3V2S1/assets/169042919/836c7c3a-14c0-46eb-983a-e7385a0b8c01)
![IMG_3995](https://github.com/mriscoc/Ender3V2S1/assets/169042919/b1168508-fc23-4ca8-aa9c-21705853336f)

**Comment by Koky05** [comment]
Maybe you have wrong FW.
 If you have Sprite Pro Extruder:
 T13 - [Marlin #13 temperature sensor](https://github.com/mriscoc/Special_Configurations/releases/tag/T13) (like Sprite Pro Extruder)
 
 Ender3V2-422-BLTUBL-**_T13_**-MPC-...
 But I thing these are only custom builds.
